spec/unit/type/exec_spec.rb

```diff
@@ -396,18 +396,28 @@ def test(command, valid)
         end
       end
 
-      context 'when timeout is exceeded on POSIX', :as_platform => :posix do
-        it 'sends a SIGTERM and raises a Puppet::Error' do
+      describe 'when timeout is exceeded' do
+        subject do
           ruby_path = Puppet::Util::Execution.ruby_path()
 
           ## Leaving this commented version in here because it fails on windows, due to what appears to be
           ##  an assumption about hash iteration order in lib/puppet/type.rb#hash2resource, where
           ##  resource[]= will overwrite the namevar with ":name" if the iteration is in the wrong order
           #sleep_exec = Puppet::Type.type(:exec).new(:name => 'exec_spec sleep command', :command => "#{ruby_path} -e 'sleep 0.02'", :timeout => '0.01')
```

Comment:
Actually, maybe we should just yank that comment block from lines 403-406. Keeping commented code up-to-date is a losing effort, and commenting *here* about a possible bug in a different module is a poor way to track possible bugs.